packages/component-header/src/components/HeaderMain/NavbarContainer/index.js

```diff
@@ -35,7 +35,7 @@ const NavbarContainer = () => {
 
   return (
     // @ts-ignore
-    <Wrapper breakpoint={breakpoint} data-testid="navigation" aria-label="Main">
+    <Wrapper breakpoint={breakpoint} data-testid="navigation" aria-label="Main Site">
```

Comment:
Jira ticket says change `Main` to `Site`, not `Main Site` 
If `Main Site` is correct, please note other changes have a case issue with `SIte`